policy-test/src/web.rs

```diff
@@ -1,26 +1,28 @@
+use k8s_openapi::apimachinery::pkg::util::intstr::IntOrString;
 use linkerd_policy_controller_k8s_api::{self as k8s};
 use maplit::{btreemap, convert_args};
 
 pub fn pod(ns: &str) -> k8s::Pod {
     k8s::Pod {
         metadata: k8s::ObjectMeta {
             namespace: Some(ns.to_string()),
-            name: Some("nginx".to_string()),
+            name: Some("web".to_string()),
             annotations: Some(convert_args!(btreemap!(
                 "linkerd.io/inject" => "enabled",
                 "config.linkerd.io/proxy-log-level" => "linkerd=trace,info",
             ))),
             labels: Some(convert_args!(btreemap!(
-                "app" => "nginx",
+                "app" => "web",
             ))),
             ..Default::default()
         },
         spec: Some(k8s::PodSpec {
             containers: vec![k8s::api::core::v1::Container {
-                name: "nginx".to_string(),
-                image: Some("docker.io/library/nginx:latest".to_string()),
+                name: "hokay".to_string(),
+                image: Some("ghcr.io/olix0r/hokay:latest".to_string()),
```

Comment:
Do we want to think about moving hokay into the Linkerd org if we're going to depend on it in tests?